### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by kayla050**
**What this PR does**:
Update Readme file.
**Which issue(s) this PR fixes**:

Fixes #

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. 

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

**Checklist**:

- [x] I have added unit/e2e tests that prove your fix is effective or that this feature works.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] I have reviewed my own code and ensured that it follows the project's style guidelines.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by kayla050** [comment]
@jasonhe258 pls take a look this readme file, thx.

File: README.md [hunk 1]
```diff
@@ -26,24 +26,25 @@ For further information, please see the [detailed introduction](./docs/detailed_
 
 ### Quick Start
 
-- For a quick try, explore CSGHub's free SaaS version at the [OpenCSG website](https://opencsg.com/models).
+- For those looking to explore, experiment with CSGHub's free SaaS version on the [OpenCSG website](https://opencsg.com/models). Refer to this [brief quick start guide](https://opencsg.com/docs/en/intro) to handle LLMs/datasets and deploy LLM applications with CSGHub SaaS interface.
```
Comment (jasonhe258) [inline README.md:29]: Better to change to "For those looking to quickly explore"?

File: README_zh.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+
+[![加入微信 Slack](https://img.shields.io/badge/wechat-join_chat-white.svg?logo=wechat&style=social)](./docs/images/wechat-assistant-new.png)
+[![加入 OpenCSG Slack](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://join.slack.com/t/opencsghq/shared_invite/zt-2fmtem7hs-s_RmMeoOIoF1qzslql2q~A)
+[![加入 OpenCSG Discord](https://img.shields.io/badge/discord-join_chat-white.svg?logo=discord&style=social)](https://discord.gg/bXnu4C9BkR)
+
+**[简体中文](README_zh.md) ∙ [日本語](README_jp.md) ∙ [한국어](README_kr.md)**
```
Comment (jasonhe258) [inline README_zh.md:6]: English version link should be shown here, instead of Chinese, due to this is Chinese version.

**Comment by jasonhe258** [comment]
@kayla050 Thanks for your great contribution! Looks good to me beside of a few minor issues.

File: README_zh.md (hunk 2 above)
Comment (jasonhe258) [reply README_zh.md:6]: Please also update other lang's Readme. Thanks.

**Comment by jasonhe258** [comment]
By the way, personally I think we can add a E2E quick guide for CSGHub beginner.

**Comment by jasonhe258** [comment]
@kayla050 LGTM. Thanks for your great contribution again!

@Rader Would you please take a look again? Thanks.

File: docs/release_notes_zh.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# 版本公告
+
+- **[2024.09.15] v0.9.0**  
+  `CSGHub` 门户迁移至 Go 版本，git 服务器支持 Gitaly 并默认启用；新增了“合集”功能、资源控制台页面；运行空间的终端微调不再需要域名；此外，还进行了其他用户体验优化和错误修复。
```
Comment (Rader) [inline docs/release_notes_zh.md:4]: 运行空间的终端微调 -》 应用空间、模型推理和微调实例的运行

File: docs/release_notes_zh.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  重要版本更新，引入独立的 `user_server` 服务，将用户、组织、令牌管理从 `CSGHub` 转移到后台服务器 `CSGHub-server`；引入独立的 `starhub_server_runner service`，以实现应用空间、模型推理和微调的统一部署；增强了资源管理；现已支持不同区域的多个 Kubernetes 集群。
+
+- **[2024.07.15] v0.7.0**  
+  重要版本更新，支持模型和数据集的“多源同步”、一键微调以及资源使用计量。
+
+- **[2024.06.21] v0.6.1**  
+  修复错误并提升用户体验。
+
+- **[2024.06.18] v0.6.0**  
+  重要版本更新，支持“专用”模型推理终端；支持 `streamlit` 空间；允许用户`收藏`仓库并为仓库设置`行业标签`；增强 git 历史和提交差异详情展示。
```
Comment (Rader) [inline docs/release_notes_zh.md:16]: 支持“专用”模型推理终端 -> 支持“专属”模型推理实例

File: docs/release_notes_zh.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+- **[2024.05.14] v0.5.0**  
+  提升空间的用户体验；自动建立仓库之间（模型、数据集、代码和空间）的“关系”；支持多文件上传。
```
Comment (Rader) [inline docs/release_notes_zh.md:19]: 空间 -》 应用空间

**Review by Rader — CHANGES_REQUESTED** [review]

**Comment by kayla050** [comment]
@Rader Modified readme, csghub saas guide and release note with your comments. Could you please review it again? Thanks!

**Review by Rader — APPROVED** [review]

**Review by hiveer — APPROVED** [review]

**Pull request merged**